### PR TITLE
Allow pytest-black to be installed on and 'enabled' on Python 2.7+.

### PR DESCRIPTION
**Pull request by jaraco**
As laid out in #33, it's useful for projects to be able to require `pytest-black`, and to enable it with `--black`, on older Pythons even though it's known not to run there, because that enables projects to adopt pytest-black for the versions that do support it and not have to worry about separate dependencies and invocation strategies depending on the environment.

This PR allows the project to be installed on Python 2.7+ and for Pythons where black is installed (currently 3.6+), allows it to run. It still collects relevant files and skips them when black is unavailable.

### CODE REVIEW

File: tests/test_black.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+pytestmark = pytest.mark.usefixtures('black_available')
+
 
 def test_help_message(testdir):
     result = testdir.runpytest("--help")
     result.stdout.fnmatch_lines(["*--black*enable format checking with black"])
 
 
-def test_fail(testdir):
+def test_fail(testdir, black_available):
```
Comment (iandotmartin) [inline tests/test_black.py]: do you still need this for each individual test if you already set it up for the module above?

File: pytest_black.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import subprocess
 import re
 import sys
+import importlib
```
[the remark this answers is not in the document]
Comment (jaraco) [reply pytest_black.py]: Yes. I'm surprised neither my linter nor the test suite caught this. Fixed in 4f3ca0d.

File: tests/test_black.py (hunk 1 above)
Comment (jaraco) [reply tests/test_black.py]: It should not be needed. That was the intention by adding the mark. Fixed in ccee75d.

**Comment by iandotmartin** [comment]
lgtm, i'll get this shipped later today

**Pull request merged**